src/inversion/inversion_utils.f90

```diff
@@ -130,7 +131,7 @@ subroutine init_diffusion(bbdif, ke, en)
                 !Define hyperviscosity:
                 K2max = max(rkxmax, rkymax) ** 2
                 wfac = one / K2max
-                visc = viscosity%prediss *  (K2max * ke /en) ** f13
+                visc = viscosity%prediss *  (K2max * te /en) ** f13
```

Comment:
Does en here now contain the addition of the integral of |gradb| ?

Reply:
Yes, it does.